modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCPrimaryKeyFinderTestSetup.java

```diff
@@ -27,6 +27,11 @@ protected JDBCPrimaryKeyFinderTestSetup(JDBCTestSetup delegate) {
     @Override
     protected final void setUpData() throws Exception {
         // kill all the data
+        try {
```

Comment:
Meh, the method should not be getting a table name, each database specific subclass must implement dropMetadataView with their case of choice (see all other drop methods, they are not getting a name as a parameter).
Follow the same pattern as dropMetadataTable.

Reply:
Done.